[ZT] Apply content strategy / clean up pages (#4826)
cloudflare/cloudflare-docs: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md

> ## Required for full Cloudflare One features

Reviewer: ```suggestion
## Required for full Cloudflare Zero Trust features
```

Reviewer: We're in the process of changing the name from Zero Trust to Cloudflare One. For now, I'd leave it as Zero Trust.

> For the majority of Cloudflare Zero Trust features to work, you need to specify a team name. Examples of Cloudflare Zero Trust features depending on the team name are [HTTP policies](/cloudflare-one/policies/filtering/http-policies/), [Browser Isolation](/cloudflare-one/policies/browser-isolation/), and [device posture](/cloudflare-one/identity/devices/).

Reviewer: ```suggestion
For the majority of Cloudflare Zero Trust features to work, you need to specify a team name. Examples of Cloudflare Zero Trust features which depend on the team name are [HTTP policies](/cloudflare-one/policies/filtering/http-policies/), [Browser Isolation](/cloudflare-one/policies/browser-isolation/), and [device posture](/cloudflare-one/identity/devices/).
```

### `organization`

> Instructs the client to register device with your organization. Registration requires authentication via an [IDP](/cloudflare-one/identity/idp-integration/) or [Service Auth](/cloudflare-one/identity/service-tokens/).

Reviewer: ```suggestion
Instructs the client to register the device with your organization. Registration requires authentication via an [IdP](/cloudflare-one/identity/idp-integration/) or [Service Auth](/cloudflare-one/identity/service-tokens/).
```

**Value Type:** `string`

**Value:** Your [team name](/cloudflare-one/glossary/#team-name).

> ## Required for DNS only policy enforcement

Reviewer: ```suggestion
## Required for DNS-only policy enforcement
```

**Value Type:** `string`

**Value:** Your [DoH subdomain](/cloudflare-one/glossary/#doh-subdomain).

## Optional fields

### `service_mode`

Allows you to choose the operational mode of the client.

**Value Type:** `string`

**Value:**

- `1dot1` &mdash; Gateway enforcement of DNS policies only through [DoH](/cloudflare-one/glossary/#doh). All other traffic is handled by your device's default mechanisms.
- `warp` &mdash; (default) All traffic sent through [Cloudflare Gateway](/cloudflare-one/glossary/#cloudflare-gateway) via our encrypted tunnel. This mode is required for features such as HTTP policies, Browser Isolation, identity-based rules, and device posture.

New service modes such as **Proxy only** are not supported as a value and must be configured in the Zero Trust dashboard.

### `onboarding`

Controls the visibility of the onboarding screens that ask the user to review the privacy policy during an application's first launch.

**Value Type:** `boolean`

**Value:**

- `false` &mdash; Screens hidden.
- `true` &mdash; (default) Screens visible.

### `switch_locked`

> Allows the user to control the connected state of the application (main toggle switch).

Reviewer: ```suggestion
Allows the user to turn off the WARP switch and disconnect the client.
```

**Value Type:** `boolean`

**Value:**

> - `false` &mdash; (default) The user is able to turn switch on/off at their discretion. When the switch is off, the user will not have the ability to reach sites protected by Access that leverage certain device posture checks.

Reviewer: ```suggestion
- `false` &mdash; (default) The user is able to turn the switch on/off at their discretion. When the switch is off, the user will not have the ability to reach sites protected by Access that leverage certain device posture checks.
```

### `auto_connect`

> If switch has been turned off by user, the client will automatically turn itself back on after the specified number of minutes. We recommend keeping this set to a very low value &mdash; usually just enough time for a user to log in to hotel or airport WiFi.

Reviewer: ```suggestion
If switch has been turned off by user, the client will automatically turn itself back on after the specified number of minutes. We recommend keeping this set to a very low value &mdash usually just enough time for a user to log in to hotel or airport WiFi.
```

**Value Type:** `integer`

**Value:**

- `0` &mdash; Allow the switch to stay in the off position indefinitely until the user turns it back on.
- `1` to `1440` &mdash; Turn switch back on automatically after the specified number of minutes.

{{<Aside>}}
This parameter replaces the old `enabled` property, which can no longer be used in conjunction with the new `switch_locked` and `auto_connect`. If you want to use these parameters, you must remove `enabled`.
{{</Aside>}}

### `support_url`

> When the WARP client is deployed via MDM, the in-app **Send Feedback** button is disabled by default. This parameter allows you to re-enable the button and direct it toward your organization.

Reviewer: ```suggestion
When the WARP client is deployed via MDM, the in-app **Send Feedback** button is disabled by default. This parameter allows you to re-enable the button and direct feedback towards your organization.
```
